deployment/security-privacy.mdx

```diff
@@ -36,3 +36,48 @@ you use [dedicated infrastructure](/deployment/dedicated-infrastructure) for pro
 We monitor the public relays for abuse and malicious activity. If we detect
 abuse, we reserve the right to block offending IP addresses or users from
 accessing the public relays.
+
+## IP Privacy and Peer-to-Peer Connections
+
+Peer-to-peer networking with direct connections and hole-punching is fundamentally
+incompatible with IP address privacy. When two endpoints establish a direct
+connection, they necessarily exchange and expose their IP addresses to each other.
+This is a core requirement of how direct peer-to-peer connections work.
```

Comment:
I think this language is a little too blunt as-is. Can we instead frame this in terms that loop in the status quo?

> any direct connection between two devices stands a high chance of being the fastest connection, but always requires IP addresses be disclosed. As with the majority of the deployed internet stack today, When two endpoints establish a direct connection, they expose and exchange their IP addresses to each other.